src/client/components/SignInButton/Button.js

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+import './Button.css';
+
+const Button = () => {
```

Comment:
This is a good start, but you need to add an `onClick` handler to the button for it to actually work as expected.

Reply:
can i install addon-actions?